netbench/netbench-cli/src/report.rs

```diff
@@ -194,6 +195,25 @@ impl Report {
                     );
                 }
 
+                {
+                    let mut y = connect_time.average();
+
+                    if !f64::is_normal(y) {
+                        y = 0.0;
+                    }
+
+                    // convert micros to seconds
+                    y /= 1_000_000.0;
```

Comment:
for handshake doesnt micro make more sense?

Reply:
it still has microsecond precision since it's a float. for vega to correctly display the order of magnitude it requires it to be in the "base" unit - i.e. seconds.